frontend/public/js/presetcontroller.js

```diff
@@ -13,7 +15,7 @@ function loadPresets() {
 		}
 		for (var t in obj.tags) {
 			if (localTags.indexOf(obj.tags[t]) === -1) {
-				localTags.push(obj.tags[t]);
+				newTag({name: obj.tags[t]});
```

Comment:
Why do you use an object with a name attribute? Can't you simply just use it without an object